sdk/monitor/azure-monitor-query/TROUBLESHOOTING.md

```diff
@@ -85,9 +86,15 @@ of Azure Monitor Query library. You can also refer to
 the [Azure Identity documentation](https://docs.microsoft.com/azure/developer/java/sdk/identity)
 for more details on the various types of credential supported in `azure-identity`.
 
+### Dependency Conflicts
```

Comment:
Thinking out loud here, but this section would be common to pretty much all TSGs. AFAIK markdown files can't include other markdown files to have a consistent common section. However there may be solutions that allow it. It would save work for future and provide consistency across the system.